src/coreclr/vm/method.hpp

```diff
@@ -2859,6 +2858,14 @@ class DynamicMethodDesc : public StoredSigMethodDesc
         LIMITED_METHOD_CONTRACT;
         _ASSERTE(IsILStub());
         return HasFlags(FlagStatic)
+            && !HasFlags(FlagIsCALLI | FlagIsDelegate)
+            && GetILStubType() == StubCLRToNativeInterop;
+    }
+    bool IsFPtrToDelegateStub() const
```

Comment:
Is this name reversed? This stub goes from delegate to fptr. 

(It is not great that we use different names for the same thing. I believe that this is called ForwardDelegateStub in other files.)

Reply:
This method may be unnecessary depending on resolution of the other feedback.